src/components/TagModal.vue

```diff
@@ -172,6 +176,9 @@ export default {
 				showError(this.t('mail', 'Tag already exists'))
 				return
 			}
+			if (displayName !== null && displayName !== '') {
```

Comment:
we dont need both checks, right? We can just write: `if (displayName !== '')` and the result is the same